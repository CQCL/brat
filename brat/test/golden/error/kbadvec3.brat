constNil :: { Bit -> Vec(Bit, 0) }
constNil = { b => cons(1, nil) }
