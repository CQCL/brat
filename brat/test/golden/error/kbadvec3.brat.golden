Error in test/golden/error/kbadvec3.brat@FC {start = Pos {line = 2, col = 19}, end = Pos {line = 2, col = 31}}:
constNil = { b => cons(1, nil) }
                  ^^^^^^^^^^^^

  Expected vector of length 0
from the type:  Vec(Bool, 0)
but got vector: [1]
of length (> 0)


